test/deprecated/withStyles_test.jsx

```diff
@@ -0,0 +1,555 @@
+import React from 'react';
```

Comment:
Same question here: is this file untouched other than the move from `test/withStyles_test.jsx → test/deprecated/withStyles_test.jsx`?